### PR TITLE
add blockquote functionality to short publication and publication pages

### PR DESCRIPTION
**Pull request by mariamiah**
#887 
### changes
- adds blockquote functionality to the short publication and publication pages
blockquote

### CODE REVIEW

**Comment by edwinmp** [comment]
Did you test on the actual pages this will affect and made sure it appears as expected? One vital feature is that the `blockquote` must take on the colour of the parent publication. This is likely working by default, but need to know that it's been tested.

**Comment by edwinmp** [comment]
Also, I have a strong feeling that this feature should come with migrations.

**Comment by mariamiah** [comment]
> Did you test on the actual pages this will affect and made sure it appears as expected (not just on the admin side)? One vital feature is that the `blockquote` must take on the colour of the parent publication. This is likely working by default, but need to know that it's been tested.

Yes, tested with the appendix, chapter, foreword, summary and short publication pages and they all appear as expected

**Comment by edwinmp** [comment]
image

This doesn't match the design ... these blockquotes are too wide ... yes, that's important ... comms reviewers are that particular, so please make sure it matches the designs in the pattern library.
I suggest u don't edit this block's template as it's already being used as it is elsewhere ... provide another template that fits this use case. https://docs.wagtail.io/en/v2.9/topics/streamfield.html#template-rendering

File: di_website/publications/blocks.py [hunk 1]
```diff
@@ -49,3 +48,16 @@ class AudioMediaStreamBlock(StreamBlock):
...
+    Custom `StructBlock` that allows render text in a blockquote element
+    """
+    text = TextBlock()
+    source = TextBlock(required=False, help_text='Who is this quote acredited to?')
+    center = BooleanBlock(default=False, required=False)
+
+    class Meta:
+        icon = 'fa-quote-right'
+        template = 'blocks/publication_blockquote.html'
+
```
Comment (edwinmp) [inline di_website/publications/blocks.py:63]: This is an entirely new block, which is redundant ... u only need to specify a different template when using the already existing block i.e `BlockQuote(template='path/to/template')`

**Review by edwinmp — CHANGES_REQUESTED** [review]

**Review by edwinmp — APPROVED** [review]
Fixed the uncentered view and added the missing migrations

**Pull request merged**